README.md

```diff
@@ -1,2 +1,31 @@
-# wg
-Coordination repository of the Game Development Working Group 
+# Rust game development working group
+
+The game development working group's main purpose is to make Rust the default language option for game developers.
+
+## What are we doing?
+
+At the very high level we have two main tasks:
+
+* To improve the experience of people using rust for game development. 
+We want game developers to have a representation in the Rust project.
+We are a bridge between them and the Rust dev team.
+
+* Share knowledge and make game engine development more accessible for people that are not engine developers.
+
+## Goals for this year
+
+* Gather a list of all the main gaps that separate us from the ideal state described in the introduction. It could be tooling gaps, a language feature, a library feature (eg. custom allocators). After this is gathered, we can start working on the biggest issues in that list. There are a few initial candidates but we encourage you to tell us where you are having issues so we can help. It may involve helping out with a low-level library
+
+* Write guides to make game development in Rust more accessible for everyone.
+
+## What we are not doing
+
+* We will not build games or a game engine as this working group. 
+* We will not be curators for "approved" game development libraries. Our purpose is to help such engines to exist, not to promote one library over another.
+
+## How can you help
```

Comment:
Is this wording similar to other groups? I don't think "you" is appropriate in the charter. Check out [WebGPU charter](https://gpuweb.github.io/admin/cg-charter.html) for comparison.

Reply:
Similar to the comment below, I will keep it in the readme and stop the charter at line 25 :)

Reply:
Moved the full charter in the docs/charter